app/assets/stylesheets/organizations.scss

```diff
@@ -26,13 +26,13 @@ img.full-width {
 }
 
 div.container div.single-map{
-    position:absolute;
-    right:6%;
-    top: 75px;
-    float:right;
-    width:200px;
-    height:160px;
-    transition: 1s;
+  position:absolute;
+  right:6%;
+  top: 75px;
+  float:right;
+  width:200px;
+  height:160px;
+  transition: 1s;
```

Comment:
Seems like the reset of the file puts a space after the `:`, please make sure to be consistent here.